blackbox/test_s3.py

```diff
@@ -103,6 +107,11 @@ def tearDownClass(cls):
         crate_node.stop()
 
     def test_copy_to_s3_copy_from_s3_roundtrip(self):
+        client = Minio('127.0.0.1:9000',
```

Comment:
Should the client be closed eventually, or is that not necessary?

Reply:
It is not necessary according to https://github.com/minio/minio-py